src/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# helper library for Sphere Lebedev Rules
+add_subdirectory(sphere_lebedev_rule)
```

Comment:
I moved the external source here as a subdirectory because I feel like it is less awkward.

Reply:
It still compiles independently though.